lapack.go

```diff
@@ -11,22 +11,57 @@ const None = 'N'
 type Job byte
 
 const (
-	All       (Job) = 'A'
-	Slim      (Job) = 'S'
-	Overwrite (Job) = 'O'
+	All       Job = 'A'
+	Slim      Job = 'S'
+	Overwrite Job = 'O'
 )
 
 type CompSV byte
 
 const (
-	Compact  (CompSV) = 'P'
-	Explicit (CompSV) = 'I'
+	Compact  CompSV = 'P'
+	Explicit CompSV = 'I'
 )
 
+// Complex128 defines the complex128 interface for the Lapack function. This interface
```

Comment:
Perhaps it is some nuance with definite articles in English, but what is _the_ Lapack function? Also below.